lib/wallet.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function(options) {
+  var EC = require('elliptic').ec;
+  var ec = new EC('secp256k1');
```

Comment:
I think it's uncompressing the public generator in order to obtain and compress the public key